app/services/filter_application_choices_for_providers.rb

```diff
@@ -17,15 +17,7 @@ def search(application_choices, candidates_name)
     def status(application_choices, statuses)
       return application_choices if statuses.blank?
 
-      filtered_application_choices = application_choices.where(status: statuses)
-
-      if statuses.include?('offer_withdrawn')
-        filtered_application_choices = filtered_application_choices.or(application_choices.offer_withdrawn)
-      elsif statuses.include?('rejected')
-        filtered_application_choices = filtered_application_choices.where(offer_withdrawn_at: nil)
-      end
-
-      filtered_application_choices
+      application_choices.where(status: statuses)
```

Comment:
🎉

Reply:
We might be able to dial down the spec for this file now that the logic is simpler

Reply:
Done!